src/stim/simulators/dem_sampler.test.cc

```diff
@@ -31,20 +31,20 @@ TEST(DemSampler, basic_sizing) {
     ASSERT_FALSE(sampler.obs_buffer.data.not_zero());
     ASSERT_FALSE(sampler.det_buffer.data.not_zero());
 
-    sampler = DemSampler(
+    DemSampler sampler2 = DemSampler(
```

Comment:
Personally I try to avoid the urge to make these sorts of small improvements to tests, because they have a nasty tendency to change what is being tested. For example, this test *could have* been checking the assignment operator but just didn't document this, and this change would have made it no longer test that.